windows/src/desktop/help/common/os.md

```diff
@@ -2,23 +2,13 @@
 title: What operating systems does Keyman support?
 ---
 
-Keyman fully supports 32-bit *and* 64-bit versions of the following
+Keyman fully supports 64-bit versions of the following
 Windows operating systems:
 
--   Windows Server 2008
-
--   Windows 7
-
--   Windows 8
-
--   Windows 8.1
-
 -   Windows 10
 
 -   Windows 11
 
```

Comment:
We need the corresponding Windows Server editions listed here

Reply:
afaict, Windows Server 2016 is end of life.
And Windows Server 2019 and Windows Server 2022 are like Windows 10.